src/modm/processing/fiber/fiber.hpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023, Niklas Hauser
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+#include "task.hpp"
+
+namespace modm
+{
+
+/// @ingroup modm_processing_fiber
```

Comment:
Documentation got lost?

```suggestion
/**
 * Fiber is a cooperative threading model consisting of an execution context (stack) and a
 * function pointer representing the entry function.
 *
 * Fibers are scheduled in a round-robin fashion.
 * ...
 *
 * @ingroup modm_processing_fiber
*/
```

Reply:
NO, THE DOG ATE IT!1!!

Reply:
![GIF: cat eating homework](https://user-images.githubusercontent.com/2820734/236681611-900f1b74-f81c-42e8-b97a-9f739d7492ad.gif)